libs/oeth/swap/src/components/ApyHeader.tsx

```diff
@@ -0,0 +1,114 @@
+import { useState } from 'react';
+
+import {
+  Box,
+  Button,
+  Menu,
+  MenuItem,
+  Paper,
+  Skeleton,
+  Stack,
+  Typography,
+} from '@mui/material';
+import { defineMessage, useIntl } from 'react-intl';
+
+import { useApiesQuery } from '../queries.generated';
+
+import type { StackProps } from '@mui/material';
+
+const trailingOptions = [
+  { label: defineMessage({ defaultMessage: '30 days trailing' }), value: 30 },
+  { label: defineMessage({ defaultMessage: '7 days trailing' }), value: 7 },
+];
+
+export const ApyHeader = (props: StackProps) => {
+  const intl = useIntl();
+  const [trailing, setTrailing] = useState(trailingOptions[0]);
+  const [anchorEl, setAnchorEl] = useState<null | HTMLElement>(null);
+  const { data: apy, isLoading: apyLoading } = useApiesQuery(
+    {
+      limit: 1,
+    },
+    {
+      select: (data) => data.apies[0],
+    },
+  );
+
+  return (
+    <Paper>
+      <Stack
+        direction="row"
+        alignItems="center"
+        justifyContent="space-between"
+        px={{ xs: 2, md: 3 }}
+        py={{ xs: 1.5, md: 2 }}
```

Comment:
I do like this style of defining breakpoints. It is sometimes easier to comprehend than when I read it in tailwind.